### ISSUE TITLE
Evangelize opportunity - hardly an issue

### ISSUE THREAD

**Issue by chris-english** [mlverse/torch#316]
I firstly apologize to you all, for what may seem nonsense, kaggle and all. Two, 3, and 4 years ago there were R implementations, notebooks, for lung segmentation in various competitions. In the present, there are none and unfortunately, I really don't know torch/torchvision well enough (let's say at all) to do justice to a muilt--image multi-label. And time is, of course, short. But looking at what I've done thus far, might you advise how I would achieve the same in torch/torchvision, so R kaggle users see a way forward. https://www.kaggle.com/cenglish/torch-r , thanks and again, apoligies.

**Comment by dfalbel** [comment]
Hi @chris-english 

I would start trying to translate an existing PyTorch example to torch. torch's API is designed to be very similar to PyTorch's so most of the times you would only need to change `.` for `$`.
Can you point to an specific notebook that builds a model in PyTorch?

**Comment by chris-english** [comment]
I was working from a journal article, https://jivp-eurasipjournals.springeropen.com/articles/10.1186/1687-5281-2014-52, Larrey-Ruiz[2014]. There aren't any code repos, but it directly addresses the Right Ventricle/Left Ventricle ratio, but overall it is more traditional segmentation. To your specific notebook ?, and more in line with a kaggle-like competition, I'll see what I can do with  https://github.com/josedolz/HyperDenseNet_pytorch, as I found the approach intriguing. But that is ambition, and while I can say the word 'stride', I have yet to wrap my head around perturbing every seventh value,

I really think this package will be a game changer, seems more direct as against keras/tensorflow in R. R6, R6, Oh! OMG, R7!

I guess more generally, regards adoption, it is best to suppose this will be among bilinguals (Python/R), but then there are the great unwashed like myself who know a certain amount of R, don't know python, can navigate their underlying OS, can build software, who are trying to up their skills into modelling. Referring them/us back to the unknown seems a lift. But everything I do in R is based on some sort of translation, within R, recognizing which code you've come across on Stackoverflow addresses your problem. Off to translate. Thank you for your interest, and most especially for this implementation.

**Comment by dfalbel** [comment]
Hi @chris-english 

Thanks very much!  And sorry for pointing you to python code. 
Here's an example of a R implementation of a resnet: https://github.com/mlverse/torchvision/blob/main/R/models-resnet.R

You can see the python equivalent code [here](https://github.com/pytorch/vision/blob/master/torchvision/models/resnet.py).

In general, you will be implementing usual R functions (`def` is the keyworkd for functions in python) and nn modules, that are created with `nn_module` in R - those are the equivalent to (`class (torch.nn.Module)`) in Python.

**Comment by chris-english** [comment]
Thanks for the clarification and guidance.
